CHANGELOG.adoc

```diff
@@ -17,13 +17,21 @@ Given a version number MAJOR.MINOR.PATCH:
 == v3.0.0 (Unreleased)
 
 === Breaking changes
-* Set minimum Terraform version to 1.0.0 (fixes #49)
+* Changed minimum Terraform version to 1.0.0 (fixes #49)
+* Deprecated `vcn_cidr`, use `vcn_cidrs` instead (list of IPv4 CIDRs).
```

Comment:
We're already making a breaking change. Let's do it cleanly. No need to carry deprecated parameters forward

Reply:
The idea here is to provide a sunset period so module users can start using the new version and enjoy the additional features without reconfiguring the module definition block.

This is similar to what service team did.

I get that we need to move forward, but here the effort to offer this seamless transition is minimal.

Would you like me to totally remove ˋvar.vcn_cidr` ?

Reply:
I would prefer we make a clean breaking change but I see your point about the announcement or lack thereof.

So, we can do 2 things:

1. We carry both vcn_cidr and vcn_cidrs in a new release of 2.x. This will then be a minor change.
2. In 3.x, we implement only vcn_cidrs.

Reply:
That being said, in 2.x, we'll also need to maintain < TF 1.0.0 syntax compatibility. I'm not sure how much is affected here.

But you're right, in future we'll make announcements on each project for breaking changes.